docs/reference/api.yml

```diff
@@ -0,0 +1,3204 @@
+---
+openapi: 3.0.1
+info:
+  title: Open Food Facts Open API
+  description: As a developer, the Open Food Facts API allows you to get information and contribute to the products database. You can create great apps to help people make better food choices and also provide data to enhance the database.
+  termsOfService: https://openweathermap.org/terms
+  contact:
+    name: Open Food Facts
+    url: https://slack-ssl-openfoodfacts.herokuapp.com/
+    email: contact@openfoodfacts.org
+  license:
+    name: License (MIT, Apache 2.0, etc)
+    url: https://opendatacommons.org/licenses/odbl/summary/index.html
+  version: "2"
+servers:
+- url: https://world.openfoodfacts.org
+  description: prod
+paths:
+  /api/v2/product/{barcode}:
+    get:
+      tags:
+      - Read Requests
+      summary: Get for a specific product by barcode
+      parameters:
+      - name: barcode
+        in: path
+        description: The barcode of the product to be fetched
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: string
+          example: "04963406"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/inline_response_200'
+components:
+  schemas:
+    inline_response_200:
+      type: object
+      properties:
+        code:
+          type: string
+        product:
+          type: object
+          properties:
+            _id:
+              type: string
+            _keywords:
+              type: array
+              items:
+                type: string
```

Comment:
What is the most important for each property is to add a description…

see https://wiki.openfoodfacts.org/API_Fields
https://wiki.openfoodfacts.org/Data_fields

Keep descriptions short, but you can put links to https://wiki.openfoodfacts.org/Data_fields (use url fragment to point to the right paragraph) and eventually add a description in the wiki if needed.

You can ask about any field you are not sure of in the slack.

Reply:
https://openfoodfacts.github.io/api-documentation/#6Understandingresponses 
The source of truth